packages/blockchain-wallet-v4-frontend/src/modals/DebitCard/OrderCard/ResidentialAddress/ResidentialAddress.tsx

```diff
@@ -253,6 +260,17 @@ const ResidentialAddress = ({
 
   const findUserAddresses = async (text: string, id?: string) => {
     if (residentialAddress) {
```

Comment:
instead of wrapping the entire thing in an if statement with if inside of it. maybe do `if (!residentialAddress) return` at the top of the function here.